p44/NOTES.md

```diff
@@ -0,0 +1,31 @@
+2024-06-07 12:57:04 AGB
+
+# Generic in macros
+
+Possible enhancement:
+
+```rust
+hashmap::<u64, u64> ! (
```

Comment:
Such syntax is not possible with declarative macros.

Reply:
ok - any alternative?

Reply:
You alredy implemented it. :)

Reply:
ok - I understood - delegate the type to the assigned variable. it'll be inferred.